amxmodx/logger.cpp

```diff
@@ -0,0 +1,1164 @@
+#include <algorithm>
```

Comment:
Please do without STL. I did not check your code, but if only for `min`/`max`, you can use `ke::Min`/`Max`.